src/main/java/com/cleo/rest/pojo/Car.java

```diff
@@ -22,9 +69,48 @@ public Builder id(UUID id) {
       car.id = id;
       return this;
     }
-    
+
+    public Builder make(String make) {
+      car.make = make;
+      return this;
+    }
+
+    public Builder model(String model) {
+      car.model = model;
+      return this;
+    }
+
+    public Builder color(String color) {
+      car.color = color;
+      return this;
+    }
+
+    public Builder year(int year) {
+      car.year = year;
+      return this;
+    }
+
     public Car build() {
       return car;
     }
   }
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
For this project it doesn't matter, but in the VersaLex code base you'd want to get rid of this extra whitespace before merging.